pandas/util/_decorators.py

```diff
@@ -307,15 +322,17 @@ def decorate(func):
 
         @wraps(func)
         def wrapper(*args, **kwargs):
-            arguments = _format_argument_list(allow_args)
             if len(args) > num_allow_args:
                 warnings.warn(
-                    msg.format(arguments=arguments),
+                    msg.format(arguments=_format_argument_list(allow_args)),
```

Comment:
is this change purely cosmetic?

(it's fine, just double-checking it's not because of some issue)

Reply:
Yeah, just cosmetic / reduce overhead